kitsune/customercare/zendesk.py

```diff
@@ -74,6 +75,19 @@ def update_user(self, user):
         # https://developer.zendesk.com/api-reference/ticketing/users/user_identities/#make-identity-primary
         return zendesk_user
 
+    def get_primary_email_identity(self, zendesk_user_id):
+        """Update the primary email of the user."""
+
+        for identity in self.client.users.identities(id=zendesk_user_id):
+            if identity.primary and identity.type == "email":
+                return identity.id
+
+    def update_primary_email(self, zendesk_user_id, email):
```

Comment:
I'd give this the a similar method signature to `update_user` like:
```python
def update_user_primary_email(self, user, email=None):
    zendesk_id = int(user.profile.zendesk_id)
    email = email or user.email
    ...
```

Reply:
This method should be simple. Gets an email and sets it in Zendesk. The control of which email is passed should be handled outside of this method.